docs/hydro/texas.rst

```diff
@@ -0,0 +1,12 @@
+Texas
++++++
+ERCOT publishes actual generation by fuel type for each 15 minute settlement interval
+(see the `fuel mix report`_ for more details). The profile of each hydro plant in the Texas Interconnection is then simply the ratio of the total generation provided by ERCOT
```

Comment:
If we're wrapping the other lines, we should wrap this one too.

Reply:
Done